todo/src/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+import ADD_NEW_TODO from '../actions/index';
+
+const initialState = {
+    input : '',
```

Comment:
your state will need to have an array to track todos, it probably doesn't need to have input on it, to be honest, you can just have the input value live in your component and pass the value up as the action payload